### PR TITLE
[TITUS-2268] Add load balancer REST API

### PR DESCRIPTION
**Pull request by gabrielhartmann**
### Description of the Change

The Load Balancer API only had a gRPC interface.  This PR adds the accompanying REST API.

### CODE REVIEW

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    @Inject
+    public LoadBalancerResource(LoadBalancerService loadBalancerService) {
+        this.loadBalancerService = loadBalancerService;
+    }
+
+    @GET
+    @ApiOperation("Find the load balancer with the specified ID")
+    @Path("/{jobId}")
+    public Response getJobLoadBalancers(@PathParam("jobId") String jobId) {
```
Comment (fabiokung) [inline titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java]: Should the return here be `GetLoadBalancersResult`? Is this not causing a `ClassCastException`?

Comment (corindwyer) [reply titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java]: Can there be multiple load balancers associated with a job? Also, this should be an error at runtime as fromSingleValueObservable would not return type `Response`.

File: titus-server-runtime/src/test/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResourceTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import static com.netflix.titus.runtime.endpoint.v3.rest.RestConstants.PAGE_SIZE_QUERY_KEY;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+
+/**
+ * Tests the {@link LoadBalancerResource} class.
+ */
+public class LoadBalancerResourceTest {
```
Comment (corindwyer) [inline titus-server-runtime/src/test/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResourceTest.java]: Missing test for `getJobLoadBalancers`?

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+public class LoadBalancerResource {
+    private final LoadBalancerService loadBalancerService;
+
+    @Inject
+    public LoadBalancerResource(LoadBalancerService loadBalancerService) {
+        this.loadBalancerService = loadBalancerService;
+    }
+
+    @GET
+    @ApiOperation("Find the load balancer with the specified ID")
```
Comment (corindwyer) [inline titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java]: balancer -> balancer(s)

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java (hunk 1 above)
Comment (gabrielhartmann) [reply titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java]: Thanks @fabiokung,  as noted by @corindwyer below I forgot to write a test which would have caught that.  I've written the test and fixed the bug.

File: titus-server-runtime/src/test/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResourceTest.java (hunk 2 above)
Comment (gabrielhartmann) [reply titus-server-runtime/src/test/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResourceTest.java]: Yes, I forgot to write the test.  Thanks for catching that.

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java (hunk 3 above)
Comment (gabrielhartmann) [reply titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java]: ACK

**Pull request merged**